tox.ini

```diff
@@ -27,7 +26,7 @@ deps =
     astropydev,build_docs: astroquery
     astropydev,build_docs: jplephem
 commands =
-    offline,astropydev,numpydev,astropy31: {env:PYTEST_COMMAND} {posargs}
```

Comment:
@nabobalis did you drop the numpydev build?

Reply:
I did